tests/suites/test_suite_ssl.function

```diff
@@ -1,6 +1,34 @@
 /* BEGIN_HEADER */
 #include <mbedtls/ssl.h>
 #include <mbedtls/ssl_internal.h>
+
+#if defined(MBEDTLS_X509_CRT_PARSE_C)
+static void pre_vrfy_fn( void *data, struct mbedtls_x509_crt *crt )
+{
+    if( crt && crt->version == 3 )
+        *(int*)data = 1;
+}
+static unsigned char pre_vrfy_buffer[2048];
+static int pre_vrfy_buffer_start = 0, pre_vrfy_buffer_end = 0;
+static int pre_vrfy_send( void *ctx, const unsigned char *buf, size_t len )
```

Comment:
I know it's test code, but we need some comments here to explain the code, at least a little.